BlazarData/src/main/resources/schema.sql

```diff
@@ -214,3 +214,8 @@ CREATE TABLE queue_items (
 
 CREATE INDEX build_state_and_timestamps on repo_builds (state, endTimestamp, startTimestamp);
 DROP INDEX build_state on repo_builds;
+
+-- changeset gchomatas:15
+ALTER TABLE branches MODIFY `branch` varchar(250) NOT NULL CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
We've already done this here: https://github.com/HubSpot/Blazar/pull/349/files#diff-ec1db078ddc713b22018f83e9ba06d89R99 I there is something else going on since this did work for a while after we made that change.